src/domain/app/i18n/fi.json

```diff
@@ -708,7 +708,8 @@
     "userMenu": {
       "ariaLabelButton": "Avaa käyttäjän valikko",
       "logout": "Kirjaudu ulos",
-      "openMyProfile": "Omat tiedot"
+      "openMyProfile": "Omat tiedot",
+      "openHelsinkiProfile": "Helsinki Profiili"
```

Comment:
```suggestion
      "openHelsinkiProfile": "Helsinki-profiili"
```
Because on https://profiili.hel.fi/login using Finnish language:
![image](https://github.com/user-attachments/assets/2f406d28-6fbf-4ee1-8ea3-e5bc6f67cc02)